assets/js/base/components/quantity-selector/style.scss

```diff
@@ -1,3 +1,60 @@
-.wc-block-quantity-selector__input {
-	width: 60px;
+@mixin reset-button {
+	border: 0;
+	padding: 0;
+	margin: 0;
+	outline: none;
```

Comment:
Should we increase `focus` styles contrast if we remove the outline? Navigating with the keyboard I felt it was difficult to know which button had focus.

![Peek 2020-01-14 14-19](https://user-images.githubusercontent.com/3616980/72347721-0bae5900-36d9-11ea-9fa0-618c739efb81.gif)

Reply:
Added